### PR TITLE
Token List Bug + Freebies

### PR DESCRIPTION
**Pull request by deadeagle63**
## Description
fixed:
- clickoutside for modals
- toast popup when saving slippage in deposit modal
- migrated to react-window for infinite list to make things easier
## Types of changes

- [ ] Technical Debt (non-breaking change which removes unused code/assets)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] The related ClickUp task has been linked to this PR
- [ ] The person creating the pull request is listed in "Assignees"
- [ ] Change requires updated documentation

### CODE REVIEW

**Comment by deadeagle63** [comment]
Cant really approve my own work but when we lift the do not merge this can go in w/e @walt-1

**Comment by deadeagle63** [comment]
Do not merge this yet, currently working on the new multi token select which requires some of these changes

**Comment by deadeagle63** [comment]
Reckon once OK'd this can be merged in

**Review by Shashank729 — APPROVED** [review]

**Pull request merged**
